src/components/ProgressBar.tsx

```diff
@@ -1,76 +1,65 @@
 import 'react-sweet-progress/lib/style.css';
-
-import React from 'react';
 // @ts-ignore
 import { Progress } from 'react-sweet-progress';
 import styled from '@emotion/styled';
+import { Flex, useMediaQuery } from '@chakra-ui/react';
 
 interface ProgressBarProps {
-  children: React.ReactNode;
+  kindOfDust: string;
   id: string;
   state: number;
 }
 
-const ProgressBar = ({ children, id, state }: ProgressBarProps) => (
-  <DustProgressWrapper id={id}>
-    <DustWrapperFlex>
-      <div>{children}</div>
-      <DustFigure>{state}</DustFigure>
-    </DustWrapperFlex>
-    <ProgressWrapper>
-      <Progress
-        percent={+state * (1 / 1000) * 100}
-        theme={{
-          success: {
-            symbol: ' ',
-            color: 'rgb(223, 105, 180)',
-          },
-          active: {
-            symbol: ' ',
-            color: '#fbc630',
-          },
-          default: {
-            symbol: ' ',
-            color: '#fbc630',
-          },
-        }}
-      />
-    </ProgressWrapper>
-  </DustProgressWrapper>
-);
-
-export default ProgressBar;
+const ProgressBar = ({ kindOfDust, id, state }: ProgressBarProps) => {
+  const percent = state * (1 / 100) * 100;
+  const percentOfDustContamination = percent > 100 ? 100 : percent;
+  const [isLargerThan768] = useMediaQuery('min-width: 768px');
 
-const DustWrapperFlex = styled.div`
-  display: flex;
-  width: 30%;
-  font-size: 3vw;
-  align-items: center;
-  font-weight: 500;
-  justify-content: space-between;
-  margin: 0 0.5rem;
-  @media only screen and (min-width: 768px) {
-    font-size: 18px;
-    margin: 0 30px;
-  }
-`;
-const DustFigure = styled.div`
-  display: flex;
-  margin-left: 2vw;
-  font-weight: 800;
-  @media only screen and (min-width: 768px) {
-    font-size: 18px;
-    margin-left: 20px;
-  }
-`;
+  return (
+    <DustProgressWrapper id={id}>
+      <Flex
+        display="flex"
+        width="30%"
+        fontSize={isLargerThan768 ? '3vw' : '18px'}
+        alignItems="center"
+        fontWeight="500"
+        justifyContent="space-between"
+        margin={isLargerThan768 ? ' 0 30px' : '0 0.5rem'}
+      >
```

Comment:
```suggestion
  return (
    <DustProgressWrapper id={id}>
      <Flex
        justifyContent="space-between"
        alignItems="center"
        width="30%"
        margin={isLargerThan768 ? '0 30px' : '0 0.5rem'}
        fontSize={isLargerThan768 ? '3vw' : '18px'}
        fontWeight="500"
      >
```
보기 좋게 순서 변경해주시면 좋을 것 같고, 단위는 rem으로 통일하는 건 어때요?